test/performance/odp_ipsecfwd.c

```diff
@@ -590,9 +631,9 @@ static parse_result_t check_options(prog_config_t *config)
 	if (config->pkt_len == 0U)
 		config->pkt_len = MIN(pool_capa.pkt.max_len, PKT_SIZE);
 
-	if (config->num_thrs <= 0 || config->num_thrs > (int)MAX_QUEUES) {
-		ODPH_ERR("Invalid CPU count: %d, (min: 1, max: %u)\n", config->num_thrs,
-			 MAX_QUEUES);
+	if (config->num_thrs <= 0 || config->num_thrs > MAX_WORKERS) {
+		ODPH_ERR("Invalid CPU count: %d (min: 1, max: %d)\n", config->num_thrs,
```

Comment:
Now that the -c option is called thread count, we could replace "CPU" with "thread" here too.